x/x.go

```diff
@@ -140,7 +140,8 @@ const (
 		"X-CSRF-Token, X-Auth-Token, X-Requested-With"
 	DgraphCostHeader = "Dgraph-TouchedUids"
 
-	DgraphVersion = 2103
+	// TODO: we should use ldflag in init.go (currently not exported) to set this version
+	DgraphVersion = 2200
```

Comment:
Maybe we should add this to the "Doing a Release" doc, basically just making sure this value is correct when the release is cut. But yeah, I agree in principle that it really should be pulling from the environment.

Reply:
+1 ^ would it be possible to pull this info from the environment at build time instead?